vars/default.yml

```diff
@@ -3,7 +3,9 @@
 
 # Put internal variables here with values for unrecognized distributions.
 
-__nbde_server_package_list: []
-__nbde_server_service: ''
+__nbde_server_packages: []
+__nbde_server_services: []
+__nbde_server_user: ""
+__nbde_server_group: ""
```

Comment:
I think you can remove this file, it does not contain anything useful. The role will then fail on unrecognized distributions, and if you do not have useful default for them, this is fine.

Reply:
Actually, as Fedora is identical to RedHat, what about putting everything into `default.yml` and getting rid of RedGHat-* and Fedora?

Reply:
Also identical to CentOS. Yeah, I could get rid of everything and have `default.yml`

Reply:
I removed those files and we have these definitions in `default.yml` now, since it's the same in the currently supported distributions.